features/apt_messages.feature

```diff
@@ -128,6 +128,21 @@ Feature: APT Messages
         Calculating upgrade...
         The following packages will be upgraded:
         """
+        When I create the file `/tmp/machine-token-overlay.json` with the following:
+        """
+        {
+            "machineTokenInfo": {
+                "contractInfo": {
+                    "effectiveTo": <now+2>
+                }
+            }
+        }
+        """
+        And I append the following on uaclient config:
+        """
+        features:
+          machine_token_overlay: "/tmp/machine-token-overlay.json"
+        """
         When I update contract to use `effectiveTo` as `days=+2`
```

Comment:
Since we're using the token overlay, do we need these "When I update contract to use `effectiveTo` as `days=+2`" lines anymore?

Reply:
Yes we need, since we will first read the `machine-token.json` file then we know that the contract is approaching expiry for us to contact the contract server and get the overlay response.

Reply:
the machine token overlay is applied at machine token read time, not at contract server request time. So I'm still not sure this line is necessary :upside_down_face:

Reply:
@CalvoM keep in mind we have 2 different test overlays, one for the machine token, another one for the contract server response

Reply:
Oh, yes I had mixed up the two overlays, thanks guys!